showcase/src/app/app.component.html

```diff
@@ -98,20 +37,105 @@ <h1>STARK</h1>
 			<a mat-list-item uiSref="demo-pretty-print" uiSrefActive="active">
 				<span matLine>Pretty Print</span>
 			</a>
-			<a mat-list-item uiSref="demo-table" uiSrefActive="active">
-				<span matLine>Table</span>
+			<a mat-list-item uiSref="demo-sidebar" uiSrefActive="active">
+				<span matLine>Sidebar</span>
 			</a>
 			<a mat-list-item uiSref="stark-header" uiSrefActive="active">
 				<span matLine>Stark header</span>
 			</a>
+			<a mat-list-item uiSref="demo-table" uiSrefActive="active">
```

Comment:
Can you move this item right after the `demo-sidebar` item? We should keep them in alphabetical order...